pkg/scheduler/framework/plugins/volumebinding/binder_test.go

```diff
@@ -334,9 +334,11 @@ func (env *testEnv) initVolumes(cachedPVs []*v1.PersistentVolume, apiPVs []*v1.P
 }
 
 func (env *testEnv) updateVolumes(t *testing.T, pvs []*v1.PersistentVolume, waitCache bool) {
-	for _, pv := range pvs {
-		if _, err := env.client.CoreV1().PersistentVolumes().Update(context.TODO(), pv, metav1.UpdateOptions{}); err != nil {
+	for i, pv := range pvs {
```

Comment:
I looked up where to call this function. IIUC. It seems it's just for update operation in test. `pvs` isn't used latter.

Reply:
just same function
````go
func (env *testEnv) updateVolumes(t *testing.T, pvs []*v1.PersistentVolume, waitCache bool) {
	for _, pv := range pvs {
		if _, err := env.client.CoreV1().PersistentVolumes().Update(context.TODO(), pv, metav1.UpdateOptions{}); err != nil {
			t.Fatalf("failed to update PV %q", pv.Name)
		}
	}
	if waitCache {
                // <------- Poll tries a condition func until it returns true, an error, or the timeout is reached.
		wait.Poll(100*time.Millisecond, 3*time.Second, func() (bool, error) {
			for _, pv := range pvs {
				obj, err := env.internalPVCache.GetAPIObj(pv.Name)
				if obj == nil || err != nil {
					return false, nil
				}
				pvInCache, ok := obj.(*v1.PersistentVolume)
				if !ok {
					return false, fmt.Errorf("PV %s invalid object", pvInCache.Name)
				}
				if versioner.CompareResourceVersion(pvInCache, pv) != 0 {
					return false, nil // <------because pv is updated, old pv will not same with pvInCache, there is always false，so we will always wait 3 second.
				}
			}
			return true, nil
		})
	}
}
````

Reply:
Oh, I see. This means that 3 seconds will be wasted anyway.
It is better to add your explanation in the description of this pr `What this PR does / why we need it:`.

Reply:
`updateVolumes` should be returning a flag indicating whether or not the update actually happened and have the caller check that and fail if it didn't; same thing with `updateClaims`; also the `waitCache` seems to always be set to true and so we can remove it.

Reply:
All finished.